certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -401,9 +401,10 @@ def _enable_redirect(self, vhost, unused_options):
         :type unused_options: Not Available
         """
         redirect_block = [[
-            ['if', '($scheme != "https")'],
-            [['return', '301 https://$host$request_uri']]
-        ]]
+            ['\n    ', 'if', ' ', '($scheme != "https") '],
+            [['\n        ', 'return', ' ', '301 https://$host$request_uri'],
+             '\n    ']
+        ], ['\n']]
```

Comment:
Question: Is there a difference as far as our parser is concerned between `['\n']` and `\n` at the end of a block?